src/uniprotkb/components/protein-data-views/SubCellViz.tsx

```diff
@@ -57,9 +68,8 @@ const attachTooltips = (
     `#${locationGroup.id}term`,
   ].join(',');
   const locationTextElements = Array.from(
-    shadowRoot.querySelectorAll(locationTextSelector)
+    document.querySelectorAll(locationTextSelector)
```

Comment:
Maybe query that on the instance of the element rather than the whole document, just in case we have multiple times this visualisation on the same page